app/posts/apply-for-teacher-training/2022-06-22-home-fee-status.md

```diff
@@ -17,17 +17,17 @@ related:
 
 ## Contents
 
-* [Overview](/apply-for-teacher-training/home-fee-status/#overview)
-* [User need](/apply-for-teacher-training/home-fee-status/#user-need)
-* [Hypothesis](/apply-for-teacher-training/home-fee-status/#hypothesis)
-* [Nationality and residency section](/apply-for-teacher-training/home-fee-status/#nationality-and-residency-section)
-* [British and Irish nationals](/apply-for-teacher-training/home-fee-status/#british-and-irish-nationals)
-* [EU nationals with EU settled status](/apply-for-teacher-training/home-fee-status/#eu-nationals-with-eu-settled-status)
-* [EU nationals with EU pre-settled status](/apply-for-teacher-training/home-fee-status/#eu-nationals-with-eu-pre-settled-status)
-* [Candidates with indefinite leave to remain or right of abode](/apply-for-teacher-training/home-fee-status/#candidates-with-indefinite-leave-to-remain-or-right-of-abode)
-* [Another immigration status](/apply-for-teacher-training/home-fee-status/#another-immigration-status)
-* [Resources](/apply-for-teacher-training/home-fee-status/#resources)
-* [Next steps](/apply-for-teacher-training/home-fee-status/#next-steps)
+- [Overview](/apply-for-teacher-training/home-fee-status/#overview)
+- [User need](/apply-for-teacher-training/home-fee-status/#user-need)
+- [Hypothesis](/apply-for-teacher-training/home-fee-status/#hypothesis)
+- [Nationality and residency section](/apply-for-teacher-training/home-fee-status/#nationality-and-residency-section)
+- [British and Irish nationals](/apply-for-teacher-training/home-fee-status/#british-and-irish-nationals)
+- [EU nationals with EU settled status](/apply-for-teacher-training/home-fee-status/#eu-nationals-with-eu-settled-status)
+- [EU nationals with EU pre-settled status](/apply-for-teacher-training/home-fee-status/#eu-nationals-with-eu-pre-settled-status)
+- [Candidates with indefinite leave to remain or right of abode](/apply-for-teacher-training/home-fee-status/#candidates-with-indefinite-leave-to-remain-or-right-of-abode)
+- [Another immigration status](/apply-for-teacher-training/home-fee-status/#another-immigration-status)
+- [Resources](/apply-for-teacher-training/home-fee-status/#resources)
+- [Next steps](/apply-for-teacher-training/home-fee-status/#next-steps)
```

Comment:
What's the reason for this change? Thought both bullet styles were fine, and if anything, the asterisks are more common? 🤷

Reply:
Pedantry

Reply:
I don't mind pedantry, but if we want consistency on this it might be better to get the existing `markdownlint` to enforce it somehow (maybe there's a rule that could be added)?  There's a GitHub workflow setup which will autofix any issues so once it's setup people don't have to think about it.